common/src/configuration/global.rs

```diff
@@ -117,6 +117,7 @@ pub struct GlobalConfig {
     pub console_wallet_notify_file: Option<PathBuf>,
     pub auto_ping_interval: u64,
     pub blocks_behind_before_considered_lagging: u64,
+    pub flood_ban_max_msg_count: usize,
```

Comment:
I believe this option should be added to the config toml files, `tari_config_example.toml` and `windows.toml`, as well.